Superlista/Views/AddCollaboratorSheetView.swift

```diff
@@ -220,7 +220,7 @@ struct CollaboratorListView: View {
                 }
                 .alert(isPresented: self.$showDeleteCollabAlert) {
                     Alert(
-                        title: Text("DeleteCollabAlertTitle \(collaborators[index].name!)"),
+                        title: Text("DeleteCollabAlertTitle \(collaborators[index].name ?? "")"),
```

Comment:
Sempre que tiver uma "", se for um nome de lista coloca "NovaLista" (ou o localizable equivalente), e quando for um nome de usuário, chamar a função getNickname()